java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1148_java.html

```diff
@@ -23,3 +23,6 @@
 }
 </pre>
 
+<h2>Exceptions</h2>
+<p>This rule ignores <code>printStackTrace</code> calls with <code>Stream</code> arguments.</p>
```

Comment:
This is slightly different than what you implemented as you kept the printStackTrace(System.err) calls as non compliant.